### PR TITLE
Added basic Discord webhook reporter.

### PR DESCRIPTION
**Pull request by croadfeldt**

### CODE REVIEW

File: share/man/man1/urlwatch.1 [hunk 1]
```diff
@@ -41,6 +41,9 @@ List telegram chats the bot is joined to
 .TP
 \fB\-\-test\-slack\fR
 Send a test notification to Slack
+.TP
+\fB\-\-test\-discord\fR
+Send a test notification to Discord
```
Comment (thp) [inline share/man/man1/urlwatch.1:46]: This is not implemented, right?

**Review by thp — CHANGES_REQUESTED** [review]
Please revert the manpage change, the rest looks good and is ready to merge.

File: share/man/man1/urlwatch.1 (hunk 1 above)
Comment (croadfeldt) [reply share/man/man1/urlwatch.1:46]: That is correct, I thought I had pulled this when I saw the generic reporter test, will reverit and push up the change. Will reply to the PR when that is done. I also see it's reporting a discord error when executing a --test-reporter, the test succeeds but it reports a discord error to the command line even though there isn't one.

Comment (croadfeldt) [reply share/man/man1/urlwatch.1]: Man page reverted and added response code for no_content to remove erroneous discord repsonse error. Ready to merge.

Comment (croadfeldt) [reply share/man/man1/urlwatch.1]: Added discord embedded support, after looking at numerous test messages in discord it became hard to identify individual reports.

**Review by thp — CHANGES_REQUESTED** [review]
Please check the Travis CI code style errors.

**Review by thp — APPROVED** [review]

**Comment by thp** [comment]
Slightly cleaned up + documented in `CHANGELOG.md` here:

https://github.com/thp/urlwatch/commit/5c44616ac6798e163bf410b855ed0ba9d558bca4

**Pull request merged**